trace/src/normalization/normalize_utils.rs

```diff
@@ -0,0 +1,192 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0. This product includes software
+// developed at Datadog (https://www.datadoghq.com/). Copyright 2023-Present
+// Datadog, Inc.
+
+use crate::errors;
+
+// DEFAULT_SPAN_NAME is the default name we assign a span if it's missing and we have no reasonable fallback
+pub const DEFAULT_SPAN_NAME: &str = "unnamed_operation";
+// DEFAULT_SERVICE_NAME is the default name we assign a service if it's missing and we have no reasonable fallback
+pub const DEFAULT_SERVICE_NAME: &str = "unnamed-service";
+
+// MAX_NAME_LEN the maximum length a name can have
+pub const MAX_NAME_LEN: i64 = 100;
+// MAX_SERVICE_LEN the maximum length a service can have
+pub const MAX_SERVICE_LEN: i64 = 100;
+
+pub const MAX_TAG_LEN: i64 = 200;
```

Comment:
A bit unusual to deal with `i64` for what appears to be a length, which is usually unsigned. Is there some particular aspect of design this stems from?

Reply:
no good reason, I'll definitely change it over